src/chronos/wTraveler/update-user-handler.ts

```diff
@@ -34,24 +58,29 @@ export function handleUpdateUser(event: UpdateUser): void {
     event.logIndex.toString();
 
   const rental = new Rental(rentalId);
-  rental.nftEntity = generateNftId(TYPE, tokenId);
+  rental.nft = generateNftId(TYPE, tokenId);
   rental.borrower = userAccount.id;
   rental.lender = nft.currentOwner;
-  rental.tokenId = tokenId;
-  rental.expirationDate = expires;
+  rental.start_date = event.block.timestamp;
+  rental.expiration_date = expires;
   rental.save();
```

Comment:
Only create a rental if `expires.gt(event.block.timestamp)`. Otherwise, the rental will be created as an expired one and should be discarded